contrib/init.d/pbs_mom.in

```diff
@@ -28,12 +29,30 @@ else
 fi
 
 pidof_pbs_mom() {
-	pid=`pidof pbs_mom`
-	if [ $? -eq 0 ]; then
-		echo $pid
-	else
-		echo "-1"
-	fi
+    pids=`pidof pbs_mom`
+    if [ $? -ne 0 ]; then
+        echo "-1"
```

Comment:
But that would be abuse of the return 0 value, since that would then indicate an issue, since a valid returned PID /= 0.